managed_vms/cloudsql/README.md

```diff
@@ -0,0 +1,29 @@
+# Python Google Cloud SQL sample for Google App Engine Managed VMs
+
+This sample demonstrates how to use [Google Cloud SQL](https://cloud.google.com/sql/) (or any other SQL server) on [Google App Engine Managed VMs](https://cloud.google.com/appengine).
+
+## Setup
+
+Before you can run or deploy the sample, you will need to do the following:
+
+1. Create a Cloud SQL instance. You can do this from the [Google Developers Console](https://console.developers.google.com) or via the [Cloud SDK](https://cloud.google.com/sdk). To create it via the SDK use the following command:
+
+        $ gcloud sql instances create [your-instance-name] \
+            --assign-ip \
+            --authorized-networks 0.0.0.0/0 \
+            --tier D0
+
+2. Create a new user and database for the application. The easiest way to do this is via the [Google Developers Console](https://console.developers.google.com/project/_/sql/instances/example-instance2/access-control/users). Alternatively, you can use MySQL tools such as the command line client or workbench.
```

Comment:
If you want to recommend using the MySQL client you have to add the `gcloud sql instances set-root-password` command to give them an initial login.